admin/settings/class-general.php

```diff
@@ -106,7 +106,7 @@ public function set_form_fields()
             'field'   => array(
                 'name'  => 'cookie_timeframe_days',
                 'label' => __('Cookie timeframe', 'lnp-alby'),
-                'description' => __('Paid articles are saved in a cookie. How many days should these cookie be valid? (default: 180)', 'lnp-alby'),
+                'description' => __('Paid articles are saved in a cookie. How many days should these cookies be valid? (default: 180)', 'lnp-alby'),
```

Comment:
it's only one. why plural?

Reply:
I cannot tell from a technical perspective. 
I should be either: "these cookies" or "the cookie"

Reply:
true. :)